spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -574,6 +576,8 @@ function onTimeoutPacket(packet: Packet) {
 }
 ```
 
+Note that interchain accounts controller modules should not execute any logic upon packet receipt, i.e. the `OnRecvPacket` callback should not be called, and in case it is called, it should simply return an error acknowledgement.
```

Comment:
I think it would be clearer to just add the OnRecvPacket pseudocode with the error acknowledgement returned.